src/utils.py

```diff
@@ -1,23 +1,26 @@
+import glob
 import logging
 import math
 import os
+import pathlib
 import random
+import subprocess
 import sys
-from itertools import chain
 from collections import Iterable
-import subprocess
+from itertools import chain
 
-import glob
-from deepsense import neptune
 import numpy as np
 import pandas as pd
 import torch
 import yaml
 from PIL import Image
 from attrdict import AttrDict
+from deepsense import neptune
 from pycocotools import mask as cocomask
-from tqdm import tqdm
 from steppy.base import BaseTransformer
+from tqdm import tqdm
+
+neptune_config_path = pathlib.Path(__file__).resolve().parents[1] / 'configs' / 'neptune_config_local.yaml'
```

Comment:
@i008 what does this one do ?

Reply:
So i found it useful whenever you dont run this from the cli. For instance you  import sutff in a notebook, this will properly resolve to your local neptune file.